src/web_app/js/peerconnectionclient.js

```diff
@@ -43,6 +43,14 @@ var PeerConnectionClient = function(params, startTime) {
   this.pc_.onsignalingstatechange = this.onSignalingStateChanged_.bind(this);
   this.pc_.oniceconnectionstatechange =
       this.onIceConnectionStateChanged_.bind(this);
+  window.dispatchEvent(new CustomEvent('pccreated', {
+    detail: {
+      pc: this,
```

Comment:
Do you really need all of `this`, will not just the `this.pc_` object do? Sample applies to the pcclosed event on line 169.